private/protocol/jsonrpc/unmarshal_test.go

```diff
@@ -1166,6 +1166,26 @@ func TestOutputService3ProtocolTestTimestampMembersCase1(t *testing.T) {
 
 }
 
+func TestOutputService3ProtocolTestTimestampMembersCase2(t *testing.T) {
```

Comment:
Adding this to the `model/protocol_tests/input/json.json` is pretty straightforward but it looks like the code generation for this requires some tweaking because it will attempt to render `` into a field which takes an int64, and will cause a compile failure because of truncating the value.

```json
  {
    "description": "Fractional Timestamp members",
    "metadata": {
      "protocol": "json"
    },
    "shapes": {
      "OutputShape": {
        "type": "structure",
        "members": {
          "TimeMember": {
            "shape": "TimeType"
          },
          "StructMember": {
            "shape": "TimeContainer"
          }
        }
      },
      "TimeType": {
        "type": "timestamp"
      },
      "TimeContainer": {
        "type": "structure",
        "members": {
          "foo": {
            "shape": "TimeType"
          }
        }
      }
    },
    "cases": [
      {
        "given": {
          "output": {
            "shape": "OutputShape"
          },
          "name": "OperationName"
        },
        "result": {
          "TimeMember": 1501917228.309,
          "StructMember": {
            "foo": 1501917228.309
          }
        },
        "response": {
          "status_code": 200,
          "headers": {},
          "body": "{\"TimeMember\": 1501917228.309, \"StructMember\": {\"foo\": 1501917228.309}}"
        }
      }
    ]
  },
```